include/exadg/operators/operator_base.cpp

```diff
@@ -742,7 +789,11 @@ OperatorBase<dim, Number, n_components>::calculate_system_matrix(SparseMatrix &
   // communicate overlapping matrix parts
   system_matrix.compress(VectorOperation::add);
 
-  if(!is_dg)
+  if(!is_dg
+#ifdef DEAL_II_WITH_TRILINOS
```

Comment:
I wonder if this would be easier to read with a second if-statement guarded by `#ifdef`?

Reply:
In my opinion it is best to completely remove this block by letting the `constraint_double` object use the function in deal.II that can put an appropriate diagonal entry into the constrained rows of the matrix. Done in commit 51a8b207